### DISCUSSION TITLE
Ошибка поиска по дереву

### DISCUSSION THREAD

**Discussion by mihail-cornev** [totumonline/totum-ru-issues-and-discussions#422] [ОШБК!] [answered]
https://youtu.be/-OCJ5Q9-zHE

Выдает ошибку про поиске в дереве. Недавно обновил тотум до посл версии.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Скорее всего в результатах поиска оказывается таблица, которая никуда не привязана и у нее пустой раздел в Раздел.

Откройте список таблиц у удостоверьтесь, что у вас все таблицы имеют привязку к дереву.
